pkg/reconciler/common/releases_test.go

```diff
@@ -25,8 +25,20 @@ import (
 	util "knative.dev/operator/pkg/reconciler/common/testing"
 )
 
+const (
+	VERSION              = "0.16.0"
+	SERVING_CORE         = "https://github.com/knative/serving/releases/download/v" + VERSION + "/serving-core.yaml"
```

Comment:
This means our tests depend on github to run.

Is there any way to fake out the actual HTTP call to keep this test hermetic?

Reply:
No. https links are just example.
URL can be any valid links locally or remotely.

Reply:
Maybe we can make this a relative `file://` URL to a file that we create in a temp dir?

Reply:
we can definitely do that. as long as the path is accessible, we can use it as the link.

Reply:
+1, I think that makes it clearer that the test doesn't depend on github.